src/jbuf/jbuf.c

```diff
@@ -153,7 +154,7 @@ static void jbuf_destructor(void *data)
  *
  * @param jbp    Pointer to returned jitter buffer
  * @param min    Minimum delay in [frames]
```

Comment:
frames --> packets. Also here

Reply:
No, because there should be at least `min` frames (`f->nf`) in the buffer to make sure the video codec can use them. The `max` must be packets, because we can't pre-calculate how many video packets are one frame (this varies a lot by codec, image size, state...).